### PR TITLE
fix: Indentation in save function

### PR DESCRIPTION
**Pull request by LightQuanta**

### CODE REVIEW

File: docs/topics/jvm/jvm-spring-boot-add-db-support.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ class MessageService(val db: JdbcTemplate) {
         Message(response.getString("id"), response.getString("text"))
     }
 
-    fun save(message: Message){
+    fun save(message: Message) {
         db.update("insert into messages values ( ?, ? )",
```
Comment (AlejandraPedroza) [inline docs/topics/jvm/jvm-spring-boot-add-db-support.md:31]:
```suggestion
        db.update(
            "insert into messages values ( ?, ? )",
```

File: docs/topics/jvm/jvm-spring-boot-add-db-support.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ class MessageService(val db: JdbcTemplate) {
...
            message.id, message.text)
```
Comment (AlejandraPedroza) [inline docs/topics/jvm/jvm-spring-boot-add-db-support.md:32]:
```suggestion
           message.id, message.text
        )
```

File: docs/topics/jvm/jvm-spring-boot-add-db-support.md [hunk 3]
```diff
@@ -117,7 +117,7 @@ class MessageService(val db: JdbcTemplate) {
         Message(response.getString("id"), response.getString("text"))
     }
 
-    fun save(message: Message){
+    fun save(message: Message) {
         val id = message.id ?: UUID.randomUUID().toString()
         db.update("insert into messages values ( ?, ? )",
```
Comment (AlejandraPedroza) [inline docs/topics/jvm/jvm-spring-boot-add-db-support.md:122]:
```suggestion
        db.update(
            "insert into messages values ( ?, ? )",
```

File: docs/topics/jvm/jvm-spring-boot-add-db-support.md [hunk 4]
```diff
@@ -117,7 +117,7 @@ class MessageService(val db: JdbcTemplate) {
...
                   id, message.text)
```
Comment (AlejandraPedroza) [inline docs/topics/jvm/jvm-spring-boot-add-db-support.md:123]:
```suggestion
                  id, message.text
            )
```

**Review by AlejandraPedroza — CHANGES_REQUESTED** [review]
Hello @LightQuanta !
Thank you so much for pointing this out. Great catches!  Triggered by your formatting suggestions, we went ahead, checked the format in the whole two code snippets, and made a couple of tweaks. We’d like to apply these changes along with your suggestions. Would you mind adding them to your commit? In that way, we can merge all the updates together. 

Thanks for your help (:

File: docs/topics/jvm/jvm-spring-boot-add-db-support.md [hunk 5]
```diff
@@ -251,7 +255,7 @@ Extend the functionality of the application to retrieve the individual messages
         fun save(message: Message) {
             val id = message.id ?: UUID.randomUUID().toString()
             db.update("insert into messages values ( ?, ? )",
```
Comment (AlejandraPedroza) [inline docs/topics/jvm/jvm-spring-boot-add-db-support.md:257]:
```suggestion
            db.update(
                "insert into messages values ( ?, ? )", 
```

File: docs/topics/jvm/jvm-spring-boot-add-db-support.md [hunk 6]
```diff
@@ -251,7 +255,7 @@ Extend the functionality of the application to retrieve the individual messages
         fun save(message: Message) {
             val id = message.id ?: UUID.randomUUID().toString()
             db.update("insert into messages values ( ?, ? )", 
-                      id, message.text)
+                id, message.text)
```
Comment (AlejandraPedroza) [inline docs/topics/jvm/jvm-spring-boot-add-db-support.md:258]:
```suggestion
                id, message.text
            )
```

**Review by AlejandraPedroza — CHANGES_REQUESTED** [review]
Hi again @LightQuanta 
Thanks for going beyond! I formatted the whole code snippet that contains the last change you made. Could you please add the format updates to your commit so that we merge all together? Thanks!

**Comment by LightQuanta** [comment]
> Hi again @LightQuanta Thanks for going beyond! I formatted the whole code snippet that contains the last change you made. Could you please add the format updates to your commit so that we merge all together? Thanks!

Done

**Review by AlejandraPedroza — APPROVED** [review]

**Pull request merged**